src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -130,7 +130,7 @@ public JsonWebToken(string header, string payload)
             if (string.IsNullOrEmpty(header))
                 throw LogHelper.LogArgumentNullException(nameof(header));
 
-            if (string.IsNullOrEmpty(payload))
+            if (payload == null)
```

Comment:
Docs on Line 117 and 127 should be updated now.

Reply:
@remyblok updated.